asab/metrics/native.py

```diff
@@ -18,6 +18,10 @@ class NativeMetrics(Service):
 
 	def __init__(self, app, metrics_svc):
 		self.MemoryGauge = metrics_svc.create_gauge("os.stat")
+
+		# injecting logging metrics into logging manager
+		logging.root.manager.LogCounter = metrics_svc.create_counter("logs", dynamic_tags=True)
```

Comment:
Let's not use dynamic tags - they have overhead (correct?)

Reply:
:( ok. it would be cool to see instantly where the logs come from.

Reply:
This metric should be fairly simple.